conda/recipes/ucxx/meta.yaml

```
@@ -53,13 +61,26 @@ outputs:
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       run_exports:
         - {{ pin_subpackage("libucxx", max_pin="x.x") }}
+      ignore_run_exports_from:
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% endif %}
     requirements:
       build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ compiler('cuda') }}
```

Think the original suggestion here needed a tweak for CUDA 11 handling, which I had missed

```suggestion
        {% if cuda_major == "11" %}
        - {{ compiler('cuda11') }} ={{ cuda_version }}
        {% else %}
        - {{ compiler('cuda') }}
        {% endif %}
```